tests/src/test/java/io/smallrye/concurrency/test/FullStackTest.java

```diff
@@ -181,6 +182,11 @@ public void after() {
     public void fullStack() {
         RestAssured.when().get("/test").then().statusCode(200).body(is("OK"));
         RestAssured.when().get("/test/async").then().statusCode(500);
+    }
+
+    @Test
+    @Ignore
+    public void fullStackAsync() {
```

Comment:
Why is this disabled? It should be passing.

Reply:
Okay, I will debug it

Reply:
I found why, it's due to your change of using `ThreadContext.TRANSACTION` for the thread context provider type, which means it's not propagated by default anymore. I've fixed it.